common/config.js

```diff
@@ -45,7 +45,7 @@ module.exports = {
     offenderAssessment: {
       url: get('OASYSAPI_ENDPOINT_URL', 'http://localhost:8081', true),
       timeout: {
-        response: get('OASYSAPII_ENDPOINT_TIMEOUT_RESPONSE', 30000, true),
+        response: get('OASYSAPI_ENDPOINT_TIMEOUT_RESPONSE', 30000, true),
```

Comment:
I"m guessing this was a typo - shout if I've changed something I shouldn't have :)

Reply:
Yeah. I guess it'll have just been defaulting 🤷‍♂